src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.skia.cs

```diff
@@ -43,7 +45,8 @@ protected override Size MeasureOverride(Size availableSize)
 		{
 			var padding = Padding;
 			var availableSizeWithoutPadding = availableSize.Subtract(padding);
-			var desiredSize = Inlines.Measure(availableSizeWithoutPadding);
+			var defaultLineHeight = (float)GetComputedLineHeight();
```

Comment:
I think the cast is not needed

```suggestion
			var defaultLineHeight = GetComputedLineHeight();
```